admin/pkg/client/tarball/tarball.go

```diff
@@ -191,14 +193,57 @@ func (b *Base) extractTarball(tarPath string) error {
 		return err
 	}
 
-	cmd := exec.Command("tar", "-C", os.TempDir(), "-zxvf", tarPath) //nolint:gosec
-	cmd.Stdout = os.Stdout
-	cmd.Stderr = os.Stderr
+	err := os.Chdir(os.TempDir())
```

Comment:
This command changes the working directory for the whole `pmm`. All commands after this one are affected.
Instead of `os.Chdir` we can use absolute paths when extracting.